### PR TITLE
Initial commit of User profile entity and appropriate doctrine metadata.

### PR DESCRIPTION
**Pull request by cooperaj**
Essentially it's a skeleton of the Profile entity and is requiring fields adding.

### CODE REVIEW

File: database/migrations/Version20161024121537_create_profile_table.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Database\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema as Schema;
+
+class Version20161024121537 extends AbstractMigration
```
Comment (geeksareforlife) [inline database/migrations/Version20161024121537_create_profile_table.php]: Classname needs to match filename

**Comment by roberthunt** [comment]
I removed the generated primary key from the profile as I see no reason it can't share the primary key of User given it's a 1:1. http://docs.doctrine-project.org/projects/doctrine-orm/en/latest/tutorials/composite-primary-keys.html#use-case-2-simple-derived-identity

**Comment by geeksareforlife** [comment]
I've merged master and updated the user.dcm.yaml file toremove conflicts

This is my first YAML, I assumed that whitespace was important but I suggest it is checked!

**Comment by cooperaj** [comment]
I've lost track. Whats happening with this PR?

**Comment by dpslwk** [comment]
i think someone was hoping your where going to add some actual profile fields

most of which would come from https://github.com/NottingHack/database/blob/master/schema/tb_members.sql i think

**Comment by cooperaj** [comment]
I'll get on that then.

**Comment by dpslwk** [comment]
OK thats the AccountManager renamed and the name field split
onto gathering some profile data during registration

**Comment by dpslwk** [comment]
OK I'm done with this now, its ready for review and merger to master, I don't want to add much more to this PR, but later we will need user/profile view and edit page

**Pull request merged**